draft-oid4vc-haip-sd-jwt-vc.md

```diff
@@ -52,6 +52,7 @@ The audience of the document is implementers that require a high level of securi
 # Terminology
 
 This specification uses the terms "Holder", "Issuer", "Verifier", and "Verifiable Credential" as defined in [@!I-D.ietf-oauth-sd-jwt-vc].
+In this document, the term "Holder" is also synonymous with "Wallet", indicating the entity that receives, stores, presents, and manages Verifiable Credentials.
```

Comment:
```suggestion
This specification uses the terms "Holder", "Issuer", "Verifier", "Wallet", and "Verifiable Credential" as defined in @!OIDF.OID4VCI] and [@!OIDF.OID4VP].
```

@tlodderstedt I am not sure why we originally pointed to sd-jwt vc for the terminology. all the terms we need are defined in oid4vc specs and I think we should reference there.